packages/webdriver/src/request/request.ts

```diff
@@ -138,9 +138,17 @@ export abstract class WebDriverRequest {
                 ...(dispatcher ? { dispatcher } : {})
             })
 
+            const rawBody = await response.text()
+            let body
+            try {
+                body = rawBody ? JSON.parse(rawBody) : {}
+            } catch {
+                body = rawBody
```

Comment:
Passing along the response as text here isn't much helpful. We should throw a specific error that contains the raw response.